modules/kubernetes/opa-gatekeeper/main.tf

```diff
@@ -118,13 +122,39 @@ resource "kubernetes_namespace" "this" {
   }
 }
 
+data "helm_template" "gatekeeper" {
+  repository   = "https://open-policy-agent.github.io/gatekeeper/charts"
+  chart        = "gatekeeper"
+  name         = "gatekeeper"
+  version      = "3.7.1"
+  include_crds = true
+}
+
+data "kubectl_file_documents" "gatekeeper" {
+  content = data.helm_template.gatekeeper.manifest
+}
+
+resource "kubectl_manifest" "gatekeeper" {
```

Comment:
Don't gatekeeper manage the crd on it's own?

Reply:
I want to manage the CRDs in the same way for all charts, even if they have their own method to update CRDs